Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.12-slim
+LABEL maintainer="m.shmygel@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    my_user
+
+RUN chown -R my_user /files/media
+RUN chmod -R 755 /files/media
+
+USER my_user
```

Comment:
Switching to a non-root user (`my_user`) is a good practice for security. Ensure that this user has the necessary permissions to execute the application and access required directories.